python/ct/crypto/verify_test.py

```diff
@@ -249,37 +249,71 @@ def test_verify_sth_consistency_invalid_proof(self):
                           verifier.verify_sth_consistency,
                           old_sth, new_sth, proof)
 
-    def test_verify_sct_valid_signature(self):
-        root_cert = cert.Certificate.from_pem_file(
-            os.path.join(FLAGS.testdata_dir, 'ca-cert.pem'))
-        test_cert = cert.Certificate.from_pem_file(
-            os.path.join(FLAGS.testdata_dir, 'test-cert.pem'))
+    def _test_verify_sct(self, proof, chain, timestamp = None):
         sct = client_pb2.SignedCertificateTimestamp()
-        tls_message.decode(read_testdata_file('test-cert.proof'), sct)
+        tls_message.decode(read_testdata_file(proof), sct)
+        if timestamp is not None:
```

Comment:
`timestamp` is a fake value used to modify the SCT to make timestamp validation fail.
The real value is loaded just above.